CHANGES.rst

```diff
@@ -127,6 +127,8 @@ Bugs fixed
   Patch by James Addison.
 * #11578: HTML Search: Order non-main index entries after other results.
   Patch by Brad King.
+* #12147: autosummary: Fix a bug that wrong suffix may be used when multiple suffixes are specified by ``source_suffix``.
+  Patch by Sutou Kouhei.
```

Comment:
```suggestion
* #12147: autosummary: Fix a bug whereby the wrong suffix may be used
  when multiple suffixes are specified by ``source_suffix``.
  Patch by Sutou Kouhei.
```

Reply:
fix lint error for line-length, and suggest some text changes

Reply:
Thanks for the suggestion!
I've applied it as-is.

(I didn't know the `:confval:`!)